zanata-war/src/main/resources/messages.properties

```diff
@@ -1263,8 +1262,8 @@ jsf.email.coordinator.ReceivedReason=You are a coordinator in the "{0}" language
 
 #------ request-to-join-language email ------
 # name, loginName, localeId, nativeName
-jsf.email.joinrequest.UserRequestingToJoin=Zanata user "{0}" with id "{1}" is requesting to join the {2} ({3}) Language Team
-jsf.email.joinrequest.AddUserInstructions=You can add {0} to the {1} team as translator using the "Add Team Member" action on the language team page and searching for "{2}".
+jsf.email.joinrequest.UserRequestingToJoin=Zanata user "{0}" with id "{1}" is requesting to join the {2} ({3}) language team
+jsf.email.joinrequest.AddUserInstructions=You can add "{0}" to the "{1}" team as translator using the "Add Team Member" action on the language team page.
```

Comment:
Do we allow multiple users to have the same display name? The username is the most reliable way to find a user, so I think we should still mention it explicitly, at least until we make a link that includes the username so that the admin does not need to search.